redis-cell.dockerfile

```diff
@@ -0,0 +1,48 @@
+#----------------------------------------------------------------------------------------------
+# This work is subject to the terms of the Redis Source Available License
+# https://redislabs.com/wp-content/uploads/2019/09/redis-source-available-license.pdf
```

Comment:
Redis Cell is actually under the MIT license: https://github.com/brandur/redis-cell/blob/master/LICENSE